apps/home/stylesheets/howitworks.styl

```diff
@@ -0,0 +1,102 @@
+.HomeHowItWorks
+  border-top 1px solid colors-gray-border
+  border-bottom 1px solid colors-gray-border
+
+  &.HomeSection__content
+    padding-bottom 4em
+    padding-left 0
+    padding-right 0
```

Comment:
PROTIP: If you have trouble remembering padding/margin shorthands: it goes clockwise: top right bottom left.

Reply:
Sigh I know, I find this thing easier to read quickly (with my small brain) but you are the boss.

Reply:
Eh, it's not like we have a Stylus-linter; no-matter-how-much-i-wish-we-did